heliopy/data/util.py

```diff
@@ -141,10 +142,17 @@ def processing_func(local_dir, local_fname, **processing_kwargs)
 
     # Loaded all the data, now filter between times
     data = timefilter(data, starttime, endtime)
+    if extension == '.cdf':
+        cdf = load(fname + extension, local_dir, '')
+        if type(units) is dict:
+            units = cdf_units(cdf, keys=units)
+        else:
+            units = cdf_units(cdf)
+        return units_attach(data, units)
+    if type(units) is coll.OrderedDict:
+        return units_attach(data, units)
     if units is None:
```

Comment:
Can this line just be a regular `else:`?

Reply:
Yep! We can do it that way.